pos_order_mgmt/models/pos_order.py

```diff
@@ -110,9 +111,12 @@ def _prepare_done_order_for_pos(self):
         for payment_line in self.payment_ids:
             payment_line = self._prepare_done_order_payment_for_pos(payment_line)
             payment_lines.append(payment_line)
+        tz = self.env.user.tz
```

Comment:
Should we follow the same defaults and fallbacks as in core?: https://github.com/odoo/odoo/blob/13.0/addons/point_of_sale/models/pos_order.py#L895

I also wonder if we could/should try to guess the real tz of the order as it was originally recorded?

Reply:
we could use the original order user_id timezone instead of the current user timezone. Its unlikely that the same sales person is working on different locations in different timezones. Yes it could be - and according to murphy's law it will be - but i think this is out of scope...